Libraries/Storage/AsyncStorage.js

```diff
@@ -154,12 +154,13 @@ const AsyncStorage = {
    */
   getAllKeys: function(
     callback?: ?(error: ?Error, keys: ?Array<string>) => void,
-  ): Promise {
+  ): Promise<?Array<string>> {
```

Comment:
keys is nullable?

Can this use `$ReadOnlyArray` instead of `Array`? `$ReadOnlyArray` is stricter so we prefer it where we can.